ai-telemetry/base/solr/secret.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: Secret
+metadata:
+  name: solr
+  labels:
+    app.kubernetes.io/instance: solr
+    app.kubernetes.io/name: solr
+    app.kubernetes.io/component: solr
+type: Opaque
+data:
+  solr-password: "ZnVjOVpWNHl3Nw=="
```

Comment:
We should remove this secret and make it an ExternalSecret I think.

Reply:
- https://github.com/OCP-on-NERC/nerc-ocp-config/pull/547 for Zookeeper was actually merged right? But we can not see it running on the cluster and nether on gitops.
- Yes, I did already implement the externalsecret but I had the secret leftover 😅